auto_cpufreq/bin/auto_cpufreq_gtk.py

```diff
@@ -1,13 +1,8 @@
 #!/usr/bin/env python3
-
-import sys
-
-sys.path.append("../")
```

Comment:
How come this was not not needed at all?

Reply:
I don't know why it's not necessary at all, but as that is still working when I remove it, I'm removing it

Reply:
When one execute `auto-cpufreq` command, it calls `/opt/auto-cpufreq/venv/bin/auto-cpufreq` file, and it import `from auto_cpufreq.bin.auto_cpufreq import main` so it seems that the root folder is auto_cpufreq